docs/reference/cat/alias.asciidoc

```diff
@@ -1,8 +1,48 @@
 [[cat-alias]]
 === cat aliases
 
-`aliases` shows information about currently configured aliases to indices
-including filter and routing infos.
+Returns information about currently configured aliases to indices, including
+filter and routing information.
+
+
+[[cat-alias-api-request]]
+==== {api-request-title}
+
+`GET /_cat/aliases/{name}`
+
+[[cat-alias-api-path-params]]
+==== {api-path-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=name]
+
+
+[[cat-alias-api-query-params]]
+==== {api-query-parms-title}
+
+// Format
```

Comment:
I'm not sure the comments help all that much given the fairly descriptive tag names.

Reply:
Thanks @nik9000. Removed with 3e43b94.